moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryStatusUpdate.js

```diff
@@ -32,200 +16,46 @@ import { getUserFullName } from "src/utils/userNames";
-          setStatusUpdateAddNew(false);
-        });
-      })
-      .catch(err => {
-        console.warn("Error: " + err);
-      });
-  };
-
-  /**
-   * Handle canceling the status update
-   */
-  const handleStatusUpdateCancel = () => {
-    // Retrieve original value
-    setStatusUpdate(getStatusUpdate("project_note"));
-    // Reset edit-mode state
-    setStatusUpdateEditable(false);
-    setStatusUpdateAddNew(false);
-  };
-
-  /**
-   * Only allow the user who wrote the status to edit it
-   */
-  const isEditableComment =
-    userId === parseInt(getStatusUpdate("added_by_user_id"));
+  const statusUpdate = data.moped_project[0].moped_proj_notes[0]?.project_note;
+  const projectName = data.moped_project[0].project_name;
+  const addedBy = data.moped_project[0].moped_proj_notes[0]?.added_by;
+  const dateCreated = makeUSExpandedFormDateFromTimeStampTZ(
+    data.moped_project[0].moped_proj_notes[0]?.date_created
+  ).toUpperCase();
 
   return (
     <Grid item xs={12} className={classes.fieldGridItem}>
       <Box display="flex" justifyContent="flex-start">
-        {/*The text field is only an input/output component only,
-                it does not track or change the state. If nothing shows,
-                make sure your comments are type id = 2*/}
-        {statusUpdateEditable && (
-          <TextField
-            fullWidth
-            id="project_note_id"
-            label="Status update"
-            value={statusUpdate}
-            onChange={handleStatusUpdateChange}
-            multiline={true}
-            rows={3}
-            InputProps={{
-              readOnly: !statusUpdateEditable,
-            }}
-          />
-        )}
-        {!statusUpdateEditable && (
+        <DashboardStatusModal
+          projectId={projectId}
+          projectName={projectName}
+          modalParent="summary"
+          statusUpdate={statusUpdate}
+          queryRefetch={refetch}
+        >
           <Box className={classes.fieldBox}>
             <Typography className={classes.fieldLabel}>
               Status update
             </Typography>
-            <Typography
-              className={classes.fieldBoxTypography}
-              onClick={isEditableComment ? handleStatusUpdateEdit : null}
-            >
-              <span className={classes.fieldLabelTextSpan}>
-                {statusUpdate || "None"}
-              </span>
-            </Typography>
             {!!statusUpdate && (
-              <Typography>
-                <span className={classes.fieldAuthor}>
-                  {getStatusUpdate("added_by")}
-                </span>
-                <span className={classes.fieldLabel}>
-                  {makeUSExpandedFormDateFromTimeStampTZ(
-                    getStatusUpdate("date_created")
-                  ).toUpperCase()}
-                </span>
-              </Typography>
+              <>
+                <Typography className={classes.fieldBoxTypography}>
```

Comment:
When the summary page loads, the console is erroring with:
```
Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
```

It looks like these `Typography` components are the culprit. They are rendering `<p>`'s  inside a `<p>` when they get passed as `children` to the `DashboardStatusModal`'s typography.

I _think_ you might be able to switch these to plain ole `<span>` ? 

And actually [this comment](https://github.com/mui/material-ui/issues/13196#issuecomment-516114150) this comment suggests you can set the typography prop `display` to `"inline"` and it will use `<span>` instead of `<p>`.

Reply:
thanks, @johnclary! setting `display` to `inline` didn't resolve the error but passing the prop `component={'span'}` in and setting all the child `Typography` components to `span` solved the issue!